### ISSUE TITLE
History: no longer possible to filter by database?

### ISSUE THREAD

**Issue by hexylena** [galaxyproject/galaxy#15470]
`database:'anoGam1'` returns nothing, and I'm not seeing an advanced filter for it, is this no longer supported? (It's fine, just need to update the tutorial.)

**Comment by dannon** [comment]
Not an intentional change.  @ahmedhamidawan can you take a look?

**Comment by ahmedhamidawan** [comment]
@dannon 
Yes, we haven't had this since we added the new history in 22.05, if I'm not wrong. This can be added as a filter (would have to configure it at the backend as well, should be straightforward), should I target 23.0 for this?

**Comment by dannon** [comment]
Yeah, I think we should add this back since it's a useful filter.

**Comment by bgruening** [comment]
Is there a way to make it a general metadata filter - and treat the genome-databse as just another metadata field? I think we want to reduce the UI dependencies on "genome" centric terms.

**Comment by ahmedhamidawan** [comment]
> @bgruening 

This is how I set this up in #15505 :

https://user-images.githubusercontent.com/78516064/216724380-bf6c89be-471b-4d81-87f3-4f78c9a552c6.mov

As you can see, at the front end, we `Filter by database`, but since at the backend the `dbkey` is taken from the `metadata.dbkey` field in `HistoryDatasetAssociation`, and then stored as `genome_build` in the history item views, we do send the filter as `genome_build:db-name` to the backend. But, as you can see, at the front end, the user can see the filter as a `database` filter

**Comment by ahmedhamidawan** [comment]
We can reopen this if we still need a generic metadata filter, we can filter by genome build/database now with the `genome_build` filter
